### ISSUE TITLE
FEATURE: Dark mode (save my eyes)

### ISSUE THREAD

**Issue by RiFi2k** [Musish/Musish#371]
Anyone else agree? I'm more than happy to put in a pull to add this.

I was honestly about to start coding a web app for musickit because I use linux on every device I own but I have a free apple music account because of work, so I canceled Spotify and realized I actually have no way to play apple music songs without spinning up a VM of windows to install itunes. Every other implementation was just so-so or missing major features, glad I found this.

Long story short, I'll try and help out on some issues because you saved me a ton of time, plus I have the email of the guy who wrote most of musickit library so we can annoy him directly if needed! Just kidding... but it may help if you ever get sandboxed or rate limited because of traffic, I heard that happened to a few people who had something similar.

**Comment by BrychanOdlum** [comment]
Definitely agree!!

But this is a duplicate of #206, so lets continue discussion there

Fwiw, I think we need to define more granular colour variables, ie sidebar_background_color instead of inheriting a more general value. I stopped work on my dark theme as I thought it'd make more sense to create more variables first would be a good idea.

Open to opinions, suggestions, and would definitely appreciate a PR!! It's a much requested topic!
